ng/src/web/pages/filters/dialog.js

```diff
@@ -23,89 +24,114 @@
 
 import React from 'react';
 
-import  _ from 'gmp/locale.js';
+import _ from 'gmp/locale.js';
 
 import Layout from '../../components/layout/layout.js';
 
 import PropTypes from '../../utils/proptypes.js';
 
-import withDialog from '../../components/dialog/withDialog.js';
+import SaveDialog from '../../components/dialog/savedialog.js';
 
 import FormGroup from '../../components/form/formgroup.js';
 import TextField from '../../components/form/textfield.js';
 import Select from '../../components/form/select.js';
 
 const FilterDialog = ({
-    comment,
-    name,
+    filter,
     term,
+    title = _('New Filter'),
     type,
     types,
-    onValueChange,
+    visible,
+    onClose,
+    onSave,
   }) => {
+  const defaults = {name: _('Unnamed'), term: ''};
+
```

Comment:
Maybe put the defaults as constant into the js module. This would avoid creating it on every render.

```jsx
const DEFAULTS = {name: _('Unnamed'), term: ''};

const FilterDialog = ({
...
```

Didn't thought about that last time.